content/applications/finance/accounting/bank/reconciliation/bank_reconciliation.rst

```diff
@@ -0,0 +1,90 @@
+===================
+Bank reconciliation
```

Comment:
So basically, the current bank_reconciliation.rst is found under reconciliation.rst, and both pages share the same H1 heading 'Bank reconciliation'. We should rethink the structure or the headings.